meetings/security/2016-12-14/minutes.md

```diff
@@ -1,19 +1,63 @@
+    * This might be because of usability, and only exposing what users need. With the addition of server use cases, this may need to change.
+  * For Cipher suites, SecureTransport uses the hex IANA value to select cipher suites whereas OpenSSL lets you use text abbreviations (corresponding to the IANA values), as well as AND/OR logic.
+* The result is that you get more capabilities on Linux from OpenSSL than you get on macOS from SecureTransport
+* Additionally BlueSSLService doesn’t work on iOS because some of the SecureTransport APIs aren’t exposed.
+  * But that's not a problem if these APIs aren't required there (ie. the user required function is provided by things like URLSession)
+
+_Use of OpenSSL vs LibreSSL:_  
+* OpenSSL is being used as it provides FIPS certification, whereas LibreSSL does not
+  * There's a number of other forks of OpenSSL, but its not believed that any of the others is FIPS certified. Additionally most of the forks are subsets of the function.
+  * Reusing OS libs also makes it easier to get FIPS certification as they are already certified on macOS and iOS and there is a certified version of OpenSSL available for Linux.
+
+_Review of BlueCryptor for Crypto:_
+
+
+* BlueCryptor attempts to do the same thing for Crypto, using CommonCrypto on iOS and macOS, and OpenSSL on Linux.
+* In CommonCrypto, the function that is missing is RSA. Similar to the issue with other file formats in BlueSSLService, RSA is there in the library, but not exposed publicly.  In OpenSSL, RSA is available and the API is similar enough to the CommonCrypto API that it could be easily exposed in BlueCryptor as well.
+
+_Keychain Services:_
+* Nothing has yet been tried for keychain services. Keychain services is great on macOS/iOS, but there is no equivalent in Linux
+* Linux just relies on certificates etc being present on the file system
+* A lot of Linux servers, particularly Java based ones, use a CMS (Certificate Management System) to manage certificates and keys.  This is similar to the services provided by the Keychain API on Apple platforms.
+* Further work is required to understand if we need to implement something here, or expect the user to deal with certificate management themselves (on a per-OS basis).
+
+_Use of BlueSSLService and BlueCryptor as a prototype implementation:_  
+* Should we use BlueSSLService and BlueCryptor as a prototype implementation, and use that to work on the API surface?
+* This seems like a reasonable approach
 
-If you'd like to add anything to the agenda ahead of time, please update the agenda document.
+### Next Steps:
+* Discuss the implemented, but not exposed, function in SecureTransport and CommonCrypto with Luke Hiesterman
+* Carry out more investigation on the need (if any) for common certificate/keychain management
+* Start to build an outline proposal for the HTTP parsing library for use as a Swift Evolution "pitch" 
```

Comment:
Isn't the HTTP parsing library being addressed by a different working group? If not, can you be more specific here e.g. whether this is looking at a lower layer socket library or the ability to add https?